packages/rocketchat-livechat/client/startup/notifyUnreadRooms.js

```diff
@@ -0,0 +1,30 @@
+Meteor.startup(function() {
+	Tracker.autorun(function() {
+		if (RocketChat.settings.get('Livechat_continuous_sound_notification_new_livechat_room')) {
+			const unreadAlertCount = ChatSubscription.find({ t: 'l', open: true, alert: true, unread: 1, disableNotifications: { $ne: true }, audioNotifications: { $ne: 'nothing'} }, { fields: { unread: 1, alert: 1, rid: 1, unreadAlert: 1 } }).count();
+			const user = RocketChat.models.Users.findOne(Meteor.userId(), {
+				fields: {
+					'settings.preferences.newRoomNotification': 1
+				}
+			});
+
+			const newRoomNotification = RocketChat.getUserPreference(user, 'newRoomNotification');
+			const [audio] = $(`audio#${ newRoomNotification }`);
+			if (audio && audio.play) {
+				if (unreadAlertCount > 0) {
+					Meteor.setTimeout(() => {
```

Comment:
@rocketchat/core does anyone know... If this isn't unset does it keep executing?  Or does the meteor wrapper here work just like normal setTimeout?